src/scss/components/_modal-box.scss

```diff
@@ -218,4 +121,54 @@ body.stopPageScroll {
 		width: 100%;
 		object-fit: contain;
 	}
+}
+
+.modal-close-button {
+	position: absolute;
+	width: 16px;
+	height: 16px;
+	top: 8px;
+	right: 8px;
+	background-color: helpers.$color-white;
+
+	@include helpers.mobile() {
+		top: 20px;
+		right: 20px;
+	}
+
+	@include helpers.laptop() {
+		top: 17px;
+		right: 17px;
+	}
+
+	&::after {
+		content: "";
+		position: absolute;
+		width: 200%;
+		height: 200%;
+		left: -50%;
+		top: -50%;
+	}
+
+	.modal-close-button__line {
+		position: absolute;
+		left: 0;
+		top: 50%;
+		width: 17px;
+		height: 1px;
+		background-color: helpers.$color-black;
+		transform: rotate(135deg);
+		transition: background-color 0.3s;
+
+		&:nth-child(1) {
+			transform: rotate(-135deg);
+		}
+	}
+
+	&:hover {
```

Comment:
1. create variable in the .modal-close-button like this `$root: &`
2. move this hover state inside `.modal-close-button__line`
3. use root variabel with interpolation like this
```scss
#{$root}:hover &{
    background-color: helpers.$color-orange;
}
```

Reply:
![image](https://github.com/IlyaDevN/Horse-club/assets/106959524/17a594f8-273e-42a9-943e-8bcee84117b8)
I did this way without using variables

Reply:
you missed step 2

Reply:
Done